src/pages/marketplace/changelog.mdx

```diff
@@ -20,6 +24,11 @@ in advance to give clients ample time to make the required changes.
 
 ## Past changes
 
+### 30 September 2025
+
+- Field `country` was added to type `Company`.
+- Mutations `AddCompany` and `UpdateCompany` accept the optional `country` input field.
```

Comment:
I would like to keep this in line with the other entries in the changelog, when other fields were added.

```suggestion
- Field `country` was added to the types `AddCompanyInput` and `UpdateCompanyInput`.
```